lib/bulk_data_test_kit/v2.0.0/client/tests/delete.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require_relative '../tags'
+require_relative '../urls'
+
+module BulkDataTestKit
+  module BulkDataV200
+    module Client
+      module Tests
```

Comment:
Not sure if this is something that should be changed, but I am thinking to be consistent with the rest of the tests in other bulk data test kit suites, there is no need to add the Tests module here